diff --git a/browser/components/preferences/main.js b/browser/components/preferences/main.js
index a8de1153312f9b524f47a526fbac00ef6403d71d..c5d93b27dcbe3823fedbf8abfc75727e3807aa8a 100644
--- a/browser/components/preferences/main.js
+++ b/browser/components/preferences/main.js
@@ -569,7 +569,7 @@ function createStartupConfig(hidden = false) {
 SettingGroupManager.registerGroups({
   defaultBrowser: createDefaultBrowserConfig(),
   startup: createStartupConfig(
-    Services.prefs.getBoolPref("browser.settings-redesign.enabled", false)
+    false
   ),
 });
 
@@ -629,7 +629,7 @@ function getBundleForLocales(newLocales) {
     ])
   );
   return new Localization(
-    ["browser/preferences/preferences.ftl", "branding/brand.ftl"],
+    ["browser/preferences/preferences.ftl", "branding/brand.ftl", "browser/preferences/zen-preferences.ftl"],
     false,
     undefined,
     locales
