diff --git a/browser/components/preferences/preferences.xhtml b/browser/components/preferences/preferences.xhtml
index bb84d15bec4008ae48324a950a4d5ba7bec1d010..48b917a3ccdcbe3d368e3fd139ab50914c0b3c7a 100644
--- a/browser/components/preferences/preferences.xhtml
+++ b/browser/components/preferences/preferences.xhtml
@@ -39,6 +39,8 @@
   <link rel="stylesheet" href="chrome://browser/content/preferences/widgets/setting-pane.css" />
   <link rel="stylesheet" href="chrome://browser/content/preferences/widgets/setting-control.css" />
 
+#include zen-preferences-links.xhtml
+
   <link rel="localization" href="branding/brand.ftl"/>
   <link rel="localization" href="browser/browser.ftl"/>
     <!-- Used by fontbuilder.js -->
@@ -121,6 +123,36 @@
         iconsrc="chrome://browser/skin/preferences/category-general.svg"
         data-l10n-id="pane-general-title">
       </html:moz-page-nav-button>
+      <html:moz-page-nav-button id="category-zen-homepage"
+        view="paneZenHomepage"
+        iconsrc="chrome://browser/skin/zen-icons/home.svg"
+        data-l10n-id="pane-zen-homepage-title">
+      </html:moz-page-nav-button>
+      <html:moz-page-nav-button id="category-zen-sidebar-layout"
+        view="paneZenSidebarLayout"
+        iconsrc="chrome://browser/skin/customize.svg"
+        data-l10n-id="pane-zen-sidebar-layout-title">
+      </html:moz-page-nav-button>
+      <html:moz-page-nav-button id="category-zen-toolbar-urlbar"
+        view="paneZenToolbarUrlbar"
+        iconsrc="chrome://browser/skin/preferences/category-search.svg"
+        data-l10n-id="pane-zen-toolbar-urlbar-title">
+      </html:moz-page-nav-button>
+      <html:moz-page-nav-button id="category-zen-spaces-tabs"
+        view="paneZenSpacesTabs"
+        iconsrc="chrome://browser/skin/window.svg"
+        data-l10n-id="pane-zen-spaces-tabs-title">
+      </html:moz-page-nav-button>
+      <html:moz-page-nav-button id="category-zen-CKS"
+        view="paneZenCKS"
+        iconsrc="chrome://browser/skin/zen-icons/brackets-curly.svg"
+        data-l10n-id="pane-zen-shortcuts-title">
+      </html:moz-page-nav-button>
+      <html:moz-page-nav-button id="category-zen-adblock"
+        view="paneZenAdblock"
+        iconsrc="chrome://browser/skin/zen-icons/tracking-protection-fill.svg"
+        data-l10n-id="pane-zen-adblock-title">
+      </html:moz-page-nav-button>
       <html:moz-page-nav-button id="category-sync"
         hidden="true"
         view="paneSync"
@@ -258,6 +280,12 @@
 #include privacy.inc.xhtml
 #include sync.inc.xhtml
 #include moreFromMozilla.inc.xhtml
+#include zenSidebarLayout.inc.xhtml
+#include zenToolbarUrlbar.inc.xhtml
+#include zenSpacesTabs.inc.xhtml
+#include zenKeyboardShortcuts.inc.xhtml
+#include zenAdblock.inc.xhtml
+#include zenHomepage.inc.xhtml
         </vbox>
       </vbox>
     </vbox>
