diff --git a/browser/branding/release/branding.nsi b/browser/branding/release/branding.nsi
index 0000000000000000000000000000000000000000..1111111111111111111111111111111111111111 100644
--- a/browser/branding/release/branding.nsi
+++ b/browser/branding/release/branding.nsi
@@ -12,9 +12,9 @@
 !define BrandFullNameInternal "Peppermint"
 !define BrandFullName         "Peppermint"
 !define CompanyName           "MintFlow Technologies"
-!define URLInfoAbout          "https://zen-browser.app"
-!define URLUpdateInfo         "https://zen-browser.app/release-notes/${AppVersion}"
-!define HelpLink              "https://github.com/zen-browser/desktop/issues"
+!define URLInfoAbout          "https://github.com/AstroDevCompany/desktop"
+!define URLUpdateInfo         "https://github.com/AstroDevCompany/desktop/releases"
+!define HelpLink              "https://github.com/AstroDevCompany/desktop/issues"
 
 
 ; The OFFICIAL define is a workaround to support different urls for Release and
@@ -24,7 +24,7 @@
 !define URLStubDownloadX86 "https://download.mozilla.org/?os=win&lang=${AB_CD}&product=firefox-latest"
 !define URLStubDownloadAMD64 "https://download.mozilla.org/?os=win64&lang=${AB_CD}&product=firefox-latest"
 !define URLStubDownloadAArch64 "https://download.mozilla.org/?os=win64-aarch64&lang=${AB_CD}&product=firefox-latest"
-!define URLManualDownload "https://zen-browser.app/download"
+!define URLManualDownload "https://github.com/AstroDevCompany/desktop/releases"
 !define URLSystemRequirements "https://www.mozilla.org/firefox/system-requirements/"
 !define Channel "stable"
 
