diff --git a/browser/locales/en-US/browser/browser.ftl b/browser/locales/en-US/browser/browser.ftl
--- a/browser/locales/en-US/browser/browser.ftl
+++ b/browser/locales/en-US/browser/browser.ftl
@@ -3,23 +3,23 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 # This is the initial default title for the browser window.
-# It gets updated based on loaded tabs or private browsing state.
+# It gets updated based on loaded tabs or Ghost mode state.
 browser-main-window-default-title = { -brand-full-name }
 
 # Note: only on macOS do we use a `-` separator between the brand name and the
-# "Private Browsing" suffix.
+# "Ghost Mode" suffix.
 browser-main-private-window-title = { PLATFORM() ->
-      [macos] { -brand-full-name } — Private Browsing
-     *[other] { -brand-full-name } Private Browsing
+      [macos] { -brand-full-name } — Ghost Mode
+     *[other] { -brand-full-name } Ghost Mode
   }
 
-# This is only used on macOS; on other OSes we use the full private window
+# This is only used on macOS; on other OSes we use the full ghost window
 # title (so including the brand name) as a suffix
-browser-main-private-suffix-for-content = Private Browsing
+browser-main-private-suffix-for-content = Ghost Mode
 
 # The non-variable portion of this MUST match the translation of
 # "PRIVATE_BROWSING_SHORTCUT_TITLE" in custom.properties
-private-browsing-shortcut-text-2 = { -brand-shortcut-name } Private Browsing
+private-browsing-shortcut-text-2 = { -brand-shortcut-name } Ghost Mode
 
 # This MUST match the translation of "BRIEF_APP_DESC" in custom.properties
 browser-shortcut-description = Fast and private web browsing
@@ -239,7 +239,7 @@
     .label = Set as Default Search Engine
     .accesskey = D
 search-one-offs-context-set-as-default-private =
-    .label = Set as Default Search Engine for Private Windows
+    .label = Set as Default Search Engine for Ghost Windows
     .accesskey = P
 
 # Search engine one-off buttons with an @alias shortcut/keyword.
@@ -355,9 +355,9 @@
 quickactions-editpdf = Open PDF editor
 quickactions-cmd-editpdf = pdf
 
-# Opens a new private browsing window
-quickactions-private2 = Open private window
-quickactions-cmd-private = private browsing
+# Opens a new Ghost mode window
+quickactions-private2 = Open ghost window
+quickactions-cmd-private = Ghost mode
 
 # Opens a SUMO article explaining how to refresh
 quickactions-refresh = Refresh { -brand-short-name }
@@ -642,14 +642,14 @@
 
 # Used for asking AI assistant chat.
 urlbar-result-action-ai-chat = Ask
-# Used when the private browsing engine differs from the default engine.
+# Used when the Ghost mode engine differs from the default engine.
 # The "with" format was chosen because the search engine name can end with
 # "Search", and we would like to avoid strings like "Search MSN Search".
 # Variables
 #  $engine (String): the name of a search engine
-urlbar-result-action-search-in-private-w-engine = Search with { $engine } in a Private Window
-# Used when the private browsing engine is the same as the default engine.
-urlbar-result-action-search-in-private = Search in a Private Window
+urlbar-result-action-search-in-private-w-engine = Search with { $engine } in a Ghost Window
+# Used when the Ghost mode engine is the same as the default engine.
+urlbar-result-action-search-in-private = Search in a Ghost Window
 # The "with" format was chosen because the search engine name can end with
 # "Search", and we would like to avoid strings like "Search MSN Search".
 # Variables
@@ -942,7 +942,7 @@
     .accesskey = i
 urlbar-view-context-menu-open-in-window2 = Open in New Window
     .accesskey = N
-urlbar-view-context-menu-open-in-private-window2 = Open in New Private Window
+urlbar-view-context-menu-open-in-private-window2 = Open in New Ghost Window
     .accesskey = P
 
 ## Labels shown above groups of urlbar results
@@ -1179,10 +1179,10 @@
   .tooltiptext = Send current tab to another device
 
 # Variables
-# $shortcut (string) - Keyboard shortcut to open a new private browsing window
+# $shortcut (string) - Keyboard shortcut to open a new Ghost mode window
 toolbar-button-new-private-window =
-  .label = New private window
-  .tooltiptext = Open a new private browsing window ({ $shortcut })
+  .label = New ghost window
+  .tooltiptext = Open a new Ghost mode window ({ $shortcut })
 
 toolbar-button-share-tab =
   .label = Share
@@ -1325,21 +1325,21 @@
     .label = Choose What I Share
     .accesskey = C
 
-# Label for the indicator shown in the private browsing window titlebar.
-private-browsing-indicator-label = Private browsing
-
-# Tooltip for the indicator shown in the private browsing window titlebar.
+# Label for the indicator shown in the Ghost mode window titlebar.
+private-browsing-indicator-label = Ghost mode
+
+# Tooltip for the indicator shown in the Ghost mode window titlebar.
 private-browsing-indicator-tooltip =
-    .tooltiptext = Private browsing
+    .tooltiptext = Ghost mode
 
-# Tooltip for the private browsing indicator button that opens the info panel.
+# Tooltip for the Ghost mode indicator button that opens the info panel.
 private-browsing-indicator-button =
-    .tooltiptext = Private browsing info
+    .tooltiptext = Ghost mode info
 
-# Title shown in the private browsing info panel.
-private-browsing-info-panel-title = You’re in a Private Window
+# Title shown in the Ghost mode info panel.
+private-browsing-info-panel-title = You’re in a Ghost Window
 
-# Body copy shown in the private browsing info panel. The learn-more link text
+# Body copy shown in the Ghost mode info panel. The learn-more link text
 # is embedded in the sentence.
 private-browsing-info-panel-description = This helps keep your browsing hidden from others on this device, but doesn’t make you invisible online. <a data-l10n-name="learn-more">Who might be able to see my activity?</a>
 
@@ -1386,13 +1386,13 @@
         Extensions
         Some extensions are disabled
 
-## Private browsing reset button
+## Ghost mode reset button
 
 reset-pbm-toolbar-button2 =
-    .label = Clear private session
-    .tooltiptext = Clear private session
-reset-pbm-panel-heading2 = Clear data and start a fresh private session?
-reset-pbm-panel-description2 = This deletes history, cookies, and all other site data without closing your Private Window.
+    .label = Clear ghost session
+    .tooltiptext = Clear ghost session
+reset-pbm-panel-heading2 = Clear data and start a fresh ghost session?
+reset-pbm-panel-description2 = This deletes history, cookies, and all other site data without closing your Ghost Window.
 reset-pbm-panel-always-ask-checkbox =
      .label = Always ask me
      .accesskey = A
@@ -1400,9 +1400,9 @@
     .label = Cancel
     .accesskey = C
 reset-pbm-panel-confirm-button2 =
-    .label = Clear private session
+    .label = Clear ghost session
     .accesskey = l
-reset-pbm-panel-complete = Private session data deleted
+reset-pbm-panel-complete = Ghost session data deleted
 
 ## Autorefresh blocker
 
@@ -1434,7 +1434,7 @@
 
 # Note: Access key is set to p to match "private" in the corresponding localized label.
 popup-notification-addon-privatebrowsing-checkbox2 =
-    .label = Allow extension to run in private windows
+    .label = Allow extension to run in ghost windows
     .accesskey = p
 
 # This string is similar to `webext-perms-description-data-long-technicalAndInteraction`
